source/objectid.rst

```diff
@@ -85,6 +85,10 @@ A 5-byte field consisting of a random value generated once per process.
 Drivers MUST NOT have an accessor method on an ObjectID class for obtaining
 this value.
 
```

Comment:
Can we also reiterate "The random value is unique to the machine and process"?

Maybe we decided against adding recommendations in a previous review, but I think a recommendation on how to generate this random number might be helpful. E.g.
"If possible, use a PRNG with OS supplied entropy that SHOULD NOT block to wait for more entropy to become available. An example is Python's random.SystemRandom. Otherwise, seed a deterministic PRNG to ensure uniqueness of process and machine by combining time, process ID, and hostname"

Reply:
I've updated the text to your suggestion, but left out the Python example.

Reply:
LGTM!